first-party/jib-quarkus-extension-gradle/README.md

```diff
@@ -0,0 +1,43 @@
+# Jib Quarkus Extension
+
+Enables containerizing a Quarkus app built with [Quarkus Gradle Plugin](https://plugins.gradle.org/plugin/io.quarkus).
+
+The Quarkus app framework prepares a special "runner" JAR and aguments dependency JARs, where the standard Jib containerization does not fit. This extension takes the JARs prepared by Quakus (runner JAR and augmented dependency JARs) and sets the entrypoint to run the runner JAR.
+
+## Examples
+
+Check out the [genenal instructions](../../README.md#using-jib-plugin-extensions) for applying a Jib plugin extension.
+
+```gradle
+// should be at the top of build.gradle
+buildscript {
+  dependencies {
+    classpath('com.google.cloud.tools:jib-quarkus-extension-gradle:0.1.0')
+  }
+}
+
+...
+
+jib {
+  ...
+  container {
+    mainClass = 'bogus'  // to suppress Jib warning about missing main class
```

Comment:
this is kinda unfortunate, could we validate that value **after** applying all extensions?

Reply:
Yeah, this is unfortunate. It's not possible with the current Jib processing. Supporting this needs some thoughts.